doc/userguide/rules/transforms.rst

```diff
@@ -106,3 +106,7 @@ Example::
 
 .. note:: depends on libnss being compiled into Suricata
 
+url_decode
+-------------------
+
+Decodes url-encoded data, ie replacing '+' with space and '%HH' with its value.
```

Comment:
Since the code mentions its not decoding unicode, would it make sense to mention that here as well?

Reply:
Ok and I take it as a confirmation that unicode decoding should not be part of this keyword